app/javascript/components/search/StudySearchProvider.js

```diff
@@ -2,9 +2,9 @@ import React, { useContext } from 'react'
 import { fetchSearch, fetchFacets, buildSearchQueryString, buildFacetsFromQueryString } from 'lib/scp-api'
 import _cloneDeep from 'lodash/cloneDeep'
 import _assign from 'lodash/assign'
-import { Router, navigate } from "@reach/router"
-import * as queryString from 'query-string';
-
+import { Router, navigate } from '@reach/router'
+import * as queryString from 'query-string'
+/* eslint-disable */
 /*
```

Comment:
why are we disabling eslint here?  Should I take an action item to clean up this file (since I was the main author)?

Reply:
ESLint did major cleanup to the point where it made major (in my opinion) edits to the code.  So feel free to if you'd like!